apps/auth/db/commands.py

```diff
@@ -52,6 +53,9 @@ def run(self, username, password, email, admin):
             user = superdesk.get_resource_service('users').find_one(username=userdata.get('username'), req=None)
 
             if user:
+                if no_update:
+                    logger.info('user already exists %s' % (userdata))
+                    return userdata
                 logger.info('updating user %s' % (userdata))
                 superdesk.get_resource_service('users').patch(user.get('_id'), userdata)
```

Comment:
Personally i would even remove updating part from that command as it's `CreateUserCommand` not `CreateOrUpdateUserCommand`. It would also fix issue from ticket.